src/elements/emby-itemscontainer/emby-itemscontainer.js

```diff
@@ -1,15 +1,17 @@
-import itemShortcuts from 'itemShortcuts';
-import inputManager from 'inputManager';
-import playbackManager from 'playbackManager';
-import imageLoader from 'imageLoader';
-import layoutManager from 'layoutManager';
-import browser from 'browser';
-import dom from 'dom';
-import loading from 'loading';
-import focusManager from 'focusManager';
-import serverNotifications from 'serverNotifications';
-import events from 'events';
-import 'webcomponents';
+
```

Comment:
should probably remove this line

Reply:
I noticed this in a few other files also, but was thinking we could probably have a lint rule cover it and automatically fix them in a follow up PR :man_shrugging: 

Relevant eslint rule: https://eslint.org/docs/rules/no-multiple-empty-lines#maxbof

Reply:
fixed new line for now